.changes/unreleased/Fixes-20230630-092618.yaml

```
@@ -0,0 +1,6 @@
+kind: Fixes
+body: Change batch_id to model override
+time: 2023-06-30T09:26:18.854492+01:00
+custom:
+  Author: nickozilla
+  Issue: "822"
```

After the revert in #826, we want to update this from a fix to a reimplementation of the original feature in #671
```suggestion
kind: Features
body: Change batch_id to model override
time: 2023-06-30T09:26:18.854492+01:00
custom:
  Author: nickozilla
  Issue: "671"
```